src/components/ebay-progress-spinner/marko-tag.json

```diff
@@ -0,0 +1,7 @@
+{
+    "@size": {
```

Comment:
You will need to add:
```
  "attribute-groups": ["html-attributes"],
  "@*": {
    "targetProperty": null,
    "type": "expression"
  },
  "@html-attributes": "expression"
```
(The only one I know for sure is needed is the `@*` one for now, not sure if the rest are needed, but for constancy, lets add them for now)